backend/controllers/homepage/homepage.ts

```diff
@@ -219,7 +219,7 @@ const getTasks = ({ activeCaseLoadId, locations, staffId, whereaboutsConfig, key
       id: 'manage-restricted-patients',
       heading: 'Manage Restricted Patients',
       description:
-        'View all restricted patients in a secure hospital, move someone to a secure hospital, and remove someone from restricted patients.',
+        'View your restricted patients, move someone to a secure hospital, or remove someone from the Restricted Patients service.',
```

Comment:
odd to see capitalisation of both Restricted and Patients, but then the heading is the same so 🤷.  I thought that our services should only be capitalised on the first letter.  Interesting also that the service heading is _Manage Restricted Patients_, but then it is referred to here as just _Restricted Patients_.

Reply:
I can pick it up with our content designer!